src/core/mesh/qgsmeshdatasetgroupstore.h

```diff
@@ -259,6 +266,8 @@ class QgsMeshDatasetGroupStore: public QObject
 
     void removePersistentProvider();
 
+    void reindexDatasetGroups();
```

Comment:
Even if private, it's worth describing what this does and when it is / should be called.